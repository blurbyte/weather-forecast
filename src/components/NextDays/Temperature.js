import styled from 'styled-components';

import media from '../../styles/media';
import Text from '../Text';

const Temperature = styled(Text).attrs({
  big: true
})`
  line-height: 1.2;

  &:first-child {
    margin-right: 1rem;
  }

  ${media.tablet`
    width: 3rem;
    text-align: right;
  `};
`;

export default Temperature;
