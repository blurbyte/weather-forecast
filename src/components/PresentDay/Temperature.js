import styled from 'styled-components';

const Temperature = styled.p`
  font-family: 'Cardo', serif;
  line-height: 1;
  font-weight: 700;
  letter-spacing: -2px;
  font-size: 8rem;
  margin: 0;
  padding-left: 1rem;
`;

export default Temperature;
