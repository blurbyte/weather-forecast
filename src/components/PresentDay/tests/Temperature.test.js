import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Temperature from '../Temperature';

test('renders correctly', () => {
  const component = renderer.create(<Temperature />);
  expect(component.toJSON()).toMatchSnapshot();
});
