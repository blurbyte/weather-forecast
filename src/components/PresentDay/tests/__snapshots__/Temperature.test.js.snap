// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  font-family: 'Cardo',serif;
  line-height: 1;
  font-weight: 700;
  -webkit-letter-spacing: -2px;
  -moz-letter-spacing: -2px;
  -ms-letter-spacing: -2px;
  letter-spacing: -2px;
  font-size: 8rem;
  margin: 0;
  padding-left: 1rem;
}

<p
  className="c0"
/>
`;
